requirements.txt

```diff
@@ -6,18 +6,18 @@ Babel~=2.12.1
 bleach==6.0.0
 bleach-allowlist==1.0.3
 blinker~=1.6.2
-boto3==1.28.1
-botocore==1.31.1
+boto3==1.28.3
+botocore==1.31.3
```

Comment:
Updated package versions since I was already in the file.

Reply:
I would prefer to not mix things that have nothing to do with each other. Makes reverting things harder and also harder to cherry-pick for forked projects.

Reply:
Makes sense to me! I'll revert these changes.